src/main/kotlin/org/elm/lang/core/psi/ElmPsiFactory.kt

```diff
@@ -24,60 +26,59 @@ class ElmPsiFactory(private val project: Project) {
-                EXPOSED_UNION_CONSTRUCTORS -> return ElmExposedUnionConstructors(node)
-                EXPOSED_UNION_CONSTRUCTOR -> return ElmExposedUnionConstructor(node)
-                EXPOSING_LIST -> return ElmExposingList(node)
-                EXPRESSION -> return ElmExpression(node)
-                FIELD -> return ElmField(node)
-                FIELD_TYPE -> return ElmFieldType(node)
-                FUNCTION_DECLARATION_LEFT -> return ElmFunctionDeclarationLeft(node)
-                GLSL_CODE -> return ElmGlslCode(node)
-                IF_ELSE -> return ElmIfElse(node)
-                IMPORT_CLAUSE -> return ElmImportClause(node)
-                INFIX_DECLARATION -> return ElmInfixDeclaration(node)
-                INNER_TYPE_ANNOTATION -> return ElmInnerTypeAnnotation(node)
-                INNER_VALUE_DECLARATION -> return ElmInnerValueDeclaration(node)
-                LET_IN -> return ElmLetIn(node)
-                LIST -> return ElmList(node)
-                LIST_OF_OPERANDS -> return ElmListOfOperands(node)
-                LOWER_PATTERN -> return ElmLowerPattern(node)
-                LOWER_TYPE_NAME -> return ElmLowerTypeName(node)
-                MODULE_DECLARATION -> return ElmModuleDeclaration(node)
-                NON_EMPTY_TUPLE -> return ElmNonEmptyTuple(node)
-                OPERATOR -> return ElmOperator(node)
-                OPERATOR_AS_FUNCTION -> return ElmOperatorAsFunction(node)
-                PARAMETRIC_TYPE_REF -> return ElmParametricTypeRef(node)
-                PARENTHESED_EXPRESSION -> return ElmParenthesedExpression(node)
-                PATTERN -> return ElmPattern(node)
-                PATTERN_AS -> return ElmPatternAs(node)
-                PORT_ANNOTATION -> return ElmPortAnnotation(node)
-                RECORD -> return ElmRecord(node)
-                RECORD_PATTERN -> return ElmRecordPattern(node)
-                RECORD_TYPE -> return ElmRecordType(node)
-                TUPLE_PATTERN -> return ElmTuplePattern(node)
-                TUPLE_TYPE -> return ElmTupleType(node)
-                TYPE_ALIAS_DECLARATION -> return ElmTypeAliasDeclaration(node)
-                TYPE_ANNOTATION -> return ElmTypeAnnotation(node)
-                TYPE_DECLARATION -> return ElmTypeDeclaration(node)
-                TYPE_REF -> return ElmTypeRef(node)
-                TYPE_VARIABLE_REF -> return ElmTypeVariableRef(node)
-                UNION_MEMBER -> return ElmUnionMember(node)
-                UNION_PATTERN -> return ElmUnionPattern(node)
-                UNIT -> return ElmUnit(node)
-                UPPER_PATH_TYPE_REF -> return ElmUpperPathTypeRef(node)
-                VALUE_DECLARATION -> return ElmValueDeclaration(node)
-                VALUE_EXPR -> return ElmValueExpr(node)
-                else -> throw AssertionError("Unknown element type: " + node.elementType)
-            }
+        fun createElement(node: ASTNode): PsiElement = when (node.elementType) {
```

Comment:
nit: can you put everything after the `=` on the next line. That's the style I've been following in this project.